### PR TITLE
fix typo in multiple files

### PR DESCRIPTION
**Pull request by thapasusheel**
This PR fix the typo in multiple files

```pseudo
compresson > compression
dimensionnal > dimensional
distibution > distribution
annoted > annotated
intergrate > integrate
```

### CODE REVIEW

File: classes/class_compressedcubemap.rst
Comment (AThousandShips) [inline classes/class_compressedcubemap.rst:1]: These changes needs to be done over [here](https://github.com/godotengine/godot), changing the generated files is not allowed, see [here](https://docs.godotengine.org/en/latest/contributing/documentation/updating_the_class_reference.html)

File: classes/class_compressedcubemaparray.rst
Comment (AThousandShips) [inline classes/class_compressedcubemaparray.rst:1]: Same

File: classes/class_texture3d.rst
Comment (AThousandShips) [inline classes/class_texture3d.rst:1]: Same

**Review by AThousandShips — CHANGES_REQUESTED** [review]

File: classes/class_texture3d.rst
Comment (AThousandShips) [reply classes/class_texture3d.rst:1]: You still left this one

**Comment by thapasusheel** [comment]
@AThousandShips  I have update as per requested. Can you review again?

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Merged. Thanks and congrats on your first merged contribution!

**Pull request merged**